nifi-registry-core/nifi-registry-docs/src/main/asciidoc/user-guide.adoc

```diff
@@ -363,3 +365,58 @@ image::remove_user_from_group.png["Remove User From Group"]
 === Other Group Level Actions
 
 Editing group names, deleting groups, adding policies to/deleting policies from groups and granting special privileges to groups follow similar procedures described earlier for corresponding user level actions.
+
+== Manage Bundles
+
+Bundles can be managed through the REST API.
+
+=== Upload Bundle
+
+A bundle can be uploaded to a bucket by making a `POST` request to the following REST end-point:
+
+  /nifi-registry-api/buckets/<bucketId>/bundles/<bundleType>
+
+Replacing `bucketId` with the id of the bucket where the bundle is being uploaded to, and `bundleType` with the type of bundle being uploaded. Currently the only supported bundle type is a NiFi Archive (NAR) which can be specified as `nifi-nar`.
+
```

Comment:
Suggest:  Replac**e** `bucketId` with the id of the bucket where the bundle is being uploaded to and `bundleType` with the type of bundle being uploaded. Currently**,** the only supported bundle type is a NiFi Archive (NAR) which can be specified as `nifi-nar`.

Reply:
Also, do you think it would be helpful to link NiFi Archive (NAR) to https://nifi.apache.org/docs/nifi-docs/html/developer-guide.html#nars ?

Reply:
In my first comment here, I tired to bold my changes.  The "e" in Replace is shown properly.  But not the comma after Currently.